lib/ReactViews/Workbench/Controls/Legend.tsx

```diff
@@ -33,11 +30,10 @@ const DISPLAYABLE_MIME_TYPES = [
 const IMAGE_URL_REGEX = /[.\/](png|jpg|jpeg|gif|svg)/i;
 
 function checkMimeType(legend: Model<LegendTraits>) {
-  if (legend.urlMimeType) {
-    return !!DISPLAYABLE_MIME_TYPES[legend.urlMimeType];
-  }
-
-  return !!legend.url?.match(IMAGE_URL_REGEX);
+  return (
+    (legend.urlMimeType && !!DISPLAYABLE_MIME_TYPES[legend.urlMimeType]) ||
+    !!legend.url?.match(IMAGE_URL_REGEX)
+  );
 }
```

Comment:
Loosen Legend `mimeType` check - so now it will treat the Legend URL as an image if the `mimeType` matches **OR** the file extension matches (previously, if `mimeType` was defined, then it wouldn't look at filetype extension)

Reply:
This is because some people use weird `mimeTypes` for legends (eg `image/png;charset=UTF-8`) - but have correct file extensions (eg `png`)

Reply:
If a Legend fails `checkMimeType` - this will be displayed

![image](https://user-images.githubusercontent.com/6187649/108316757-718cde80-7211-11eb-846f-9610f500a5f6.png)